.ci/build-app-image.sh

```diff
@@ -17,7 +17,11 @@ if [[ ! -d $SAMPLES_REPO/samples/$VERSION/$TEST_APP ]]; then
     exit 1
 fi
 cd  $SAMPLES_REPO/samples/$VERSION/$TEST_APP
-tee Dockerfile << EOF
+
+if [[ -f "Dockerfile" ]]; then
```

Comment:
@kichalla I mean if the image prints the error, the CI collect container logs as well... So that should be fine.